cyclonedx/cyclonedx_test.go

```diff
@@ -24,11 +24,11 @@ import (
 	"github.com/package-url/packageurl-go"
 	"github.com/shopspring/decimal"
 	"github.com/sonatype-nexus-community/nancy/types"
-	assert "gopkg.in/go-playground/assert.v1"
+	"gopkg.in/go-playground/assert.v1"
 )
 
 func TestCreateSBOMFromPackageURLs(t *testing.T) {
-	results := []packageurl.PackageURL{}
+	var results []packageurl.PackageURL
```

Comment:
Are these linter changes?

Reply:
Yes, I was just going through linter suggestions. Happy to revert if desired.

Reply:
Naw no big deal, just kinda curious why it happened.